amundsen_application/static/js/components/TableDetail/index.tsx

```diff
@@ -142,11 +143,10 @@ class TableDetail extends React.Component<TableDetailProps & RouteComponentProps
               </h3>
               <BookmarkIcon bookmarkKey={ this.props.tableData.key }/>
               <div className="body-2">
-                <ul className="header-bullets">
-                  <li>{ getDisplayNameByResource(ResourceType.table) }</li>
-                  <li>{ getDatabaseDisplayName(data.database) }</li>
-                  <li>{ data.cluster }</li>
-                </ul>
+                <TableHeaderBullets
+                  database={ data.database }
+                  cluster={ data.cluster }
+                />
```

Comment:
Just a thought about overwriting the TableHeaderBullets -- If we wanted to add another field from the table metadata, we'd have to overwrite the component file and the template that uses the component. On the other hand, I can see how extracting specific fields and passing it to the component is cleaner and easier to test.

Reply:
I can see that. As our use cases for overwriting components internally expose themselves, I will lean towards whatever makes maintenance the least painful.